ReadMe.md

```diff
@@ -91,6 +91,7 @@ If you like or are using this project to learn or start your solution, please gi
    2. Log Management System
       - [Sentry.io](http://sentry.io)
       - [Loggly.com](https://loggly.com)
+      - [Kibana](https://www.elastic.co/products/kibana)
```

Comment:
Kibana is not Log management system, its visualizer for elasticsearch data source, of course if we use elasticsearch as logger which is obviously possible we can only visualize its data, create charts, etc. so I think it is not reasonable to add Kibana as Log Management System, please take care of that and create another PR!